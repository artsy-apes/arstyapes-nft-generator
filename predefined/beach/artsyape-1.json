{
        "body": "Lou",
        "head": "Lou",
        "jewelry": "None",
        "outfit": "None",
        "eye": "Green Eyebrows, Orange Eyes",
        "mouth attributes": "None",
        "background": "Sunny Beach",
        "glasses": "Flower Power",
        "headwear": "None"
}
