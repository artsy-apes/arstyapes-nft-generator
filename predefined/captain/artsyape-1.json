{
        "body": "Ham",
        "head": "Ham",
        "jewelry": "Golden Earring",
        "outfit": "Captain_s Outfit",
        "eye": "Grey Eyebrows, Yellow Eyes",
        "mouth attributes": "Cigar",
        "background": "Sunny Beach",
        "glasses": "None",
        "headwear": "Captains Hat"
}
