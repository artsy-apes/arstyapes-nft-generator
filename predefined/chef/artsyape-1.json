{
        "body": "Robin",
        "head": "Robin",
        "jewelry": "None",
        "outfit": "Chef_s Jacket",
        "eye": "Yellow Eyebrows, Blue Eyes",
        "mouth attributes": "Pizza",
        "background": "Deep Blue",
        "glasses": "None",
        "headwear": "Chef_s Hat"
}
