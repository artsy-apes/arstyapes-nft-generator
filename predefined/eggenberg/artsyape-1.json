{
        "body": "Albert",
        "head": "Albert",
        "jewelry": "Bitcoin Chain",
        "outfit": "Das Eggenberg Tee",
        "eye": "Brown Eyebrows, Orange Eyes",
        "mouth attributes": "None",
        "background": "Brick Wall",
        "glasses": "Pilot Sunglasses",
        "headwear": "Chef_s Hat"
}
