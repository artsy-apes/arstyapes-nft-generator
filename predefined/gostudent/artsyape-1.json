{
        "body": "Robin",
        "head": "Robin",
        "jewelry": "None",
        "outfit": "Go Student Tee",
        "eye": "Grey And Yellow",
        "mouth attributes": "None",
        "background": "Mountains",
        "glasses": "Clear Lenses Round",
        "headwear": "None"
}
