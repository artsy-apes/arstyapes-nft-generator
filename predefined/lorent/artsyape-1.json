{
        "body": "Whitney",
        "head": "Whitney",
        "jewelry": "Bitcoin Chain",
        "outfit": "Laurent Tee",
        "eye": "Orange Eyebrows, Blue Eyes",
        "mouth attributes": "Lollypop",
        "background": "Sunny Beach",
        "glasses": "Heart Sunglasses",
        "headwear": "None"
}
