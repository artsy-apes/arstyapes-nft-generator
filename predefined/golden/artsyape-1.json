{
        "body": "Golden",
        "head": "Golden",
        "jewelry": "#WAGMI Chain",
        "outfit": "Black tee",
        "eye": "None",
        "mouth attributes": "None",
        "background": "Gold Flakes",
        "glasses": "None",
        "headwear": "Gold Halo"
}
